src/main/java/ch/saunah/saunahbackend/controller/PriceController.java

```diff
@@ -0,0 +1,43 @@
+package ch.saunah.saunahbackend.controller;
+
+import ch.saunah.saunahbackend.dto.PriceBody;
+import ch.saunah.saunahbackend.dto.PriceResponse;
+import ch.saunah.saunahbackend.service.PriceService;
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Controls the different operations that can be done with price
+ */
+@RestController
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+public class PriceController {
+
+    @Autowired
+    private PriceService priceService;
+
```

Comment:
add 2 more apis, one for returning all prices and one for returning one price

Reply:
Done